rllib/algorithms/r2d2/r2d2_tf_policy.py

```diff
@@ -207,7 +207,7 @@ def h_function(x, epsilon=1.0):
 
 
 def h_inverse(x, epsilon=1.0):
-    """Inverse if the above h-function, described in the paper [1].
+    """Inverse of the above h-function, described in the paper [1].
```

Comment:
this change looks completely unrelated?

Reply:
This is to trigger library tests, since CI did not kick off library tests if no changes to libraries where made. This might have changed because I reported to Kai that we should trigger tests if requirements files change. I'll have another look.